packages/application-config/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@commercetools-frontend/application-config",
+  "version": "1.0.0",
+  "description": "Configuration utilities for building Custom Applications",
+  "bugs": "https://github.com/commercetools/merchant-center-application-kit/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/merchant-center-application-kit.git",
+    "directory": "packages/application-config"
+  },
+  "homepage": "https://docs.commercetools.com/custom-applications",
+  "keywords": [
+    "javascript",
+    "frontend",
+    "react",
+    "toolkit",
+    "config"
+  ],
+  "license": "MIT",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "./build/index.js",
+  "module": "./build/index.js",
+  "typings": "./build/typings/src/index.d.ts",
+  "types": "./build/typings/src/index.d.ts",
+  "files": [
+    "build",
+    "schema.json",
+    "package.json",
+    "LICENSE",
+    "README.md"
+  ],
+  "scripts": {
+    "prepare": "./../../scripts/version.js replace",
+    "prebuild": "rimraf build/**",
+    "build": "yarn build:schema && yarn build:src && yarn build:typings",
+    "build:schema": "json2ts schema.json src/schema.ts --style.singleQuote --bannerComment '/* eslint-disable prettier/prettier */\n// This file was automatically generated by json-schema-to-typescript.\n// DO NOT MODIFY IT BY HAND. Instead, modify the source schema.json file.'",
+    "build:src": "babel src --out-dir build --extensions .ts --ignore src/types.ts,src/schema.ts",
```

Comment:
Note: we're transpiling with Babel here as we want to retain the single files, as opposed to Rollup where we get one single bundle file.